packages/dd-trace/test/appsec/iast/vulnerability-reporter.spec.js

```diff
@@ -95,6 +95,11 @@ describe('vulnerability-reporter', () => {
         start({
           iast: {
             deduplicationEnabled: true
+          },
+          appsec: {
+            standalone: {
+              enabled: null
```

Comment:
is `enabled: null` possible? `undefined` by default, `true` or `false` if they are set, isn't it?